helm/charts/determined/templates/master-config.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+   name: determined-master-config-{{ .Release.Name }}
+   namespace: {{ .Release.Namespace }}
+   labels:
+     app: deteremined-master-{{ .Release.Name }}
+     release: {{ .Release.Name }}
+data:
+  master.yaml: |
+    checkpoint_storage:
+      type: {{ required "A valid Values.checkpointStorage.type entry is required!" .Values.checkpointStorage.type | quote}}
+      {{- if eq .Values.checkpointStorage.type "shared_fs" }}
+      host_path: {{ required "A valid Values.checkpointStorage.hostPath entry is required!" .Values.checkpointStorage.hostPath | quote }}
+      {{- else if eq .Values.checkpointStorage.type "gcs" }}
+      bucket: {{ required "A valid Values.checkpointStorage.bucket entry is required!" .Values.checkpointStorage.bucket }} 
+      {{- else if eq .Values.checkpointStorage.type "s3" }}
+      bucket: {{ required "A valid Values.checkpointStorage.bucket entry is required!" .Values.checkpointStorage.bucket }} 
+      access_key: {{ .Values.checkpointStorage.accessKey | quote }}
+      secret_key: {{ .Values.checkpointStorage.secretKey | quote }}
+      endpoint_url: {{ .Values.checkpointStorage.endpointUrl | quote }}
+      {{- end }}
+      save_experiment_best: 0
```

Comment:
non-blocking: In general I don't think users have configured the checkpoint gc to be anything other than the default, however it would be nice to still make it configurable.

Reply:
Good idea, made this configurable.